clang/test/CodeGenSYCL/builtin-alloca.cpp

```diff
@@ -0,0 +1,86 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -disable-llvm-passes -emit-llvm -o - %s \
+// RUN: | FileCheck %s --check-prefix=CHECK-DEVICE
+// RUN: %clang_cc1 -fsycl-is-host   -triple x86_64-unknown-unknown -disable-llvm-passes -emit-llvm -o - %s -fcxx-exceptions\
+// RUN: | FileCheck %s --check-prefix=CHECK-HOST
+
+#include <stddef.h>
+
+#include "Inputs/sycl.hpp"
+#include "Inputs/private_alloca.hpp"
+
+// expected-no-diagnostics
+
+struct myStruct {
+  char a;
+  char b;
+};
+
+constexpr sycl::specialization_id<size_t> size(1);
+constexpr sycl::specialization_id<int> intSize(-1);
+constexpr sycl::specialization_id<unsigned short> shortSize(1);
+
+// COM: On the device, for each call, we should generate a chain of: 'call @sycl.alloca.<ty>' + ('addrspacecast') + 'store'.
```

Comment:
```suggestion
// On the device, for each call, we should generate a chain of: 'call @sycl.alloca.<ty>' + ('addrspacecast') + 'store'.
```
Likewise for all comments below; don't need the COM: prefix.